service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerAboutToSubmitTest.java

```diff
@@ -57,15 +61,35 @@ void shouldReturnSuccessfulResponseWithValidCaseData() {
             .willReturn(pdf);
         given(uploadDocumentService.uploadPDF(userId, userAuthToken, pdf, "2313.pdf"))
             .willReturn(document);
+        given(featureToggleService.isCtscEnabled()).willReturn(false);
 
         AboutToStartOrSubmitCallbackResponse callbackResponse = postAboutToSubmitEvent("fixtures/case.json");
 
         assertThat(callbackResponse.getData())
             .containsEntry("caseLocalAuthority", "example")
+            .containsEntry("sendToCtsc", "No")
             .containsEntry("submittedForm", ImmutableMap.<String, String>builder()
                 .put("document_url", document.links.self.href)
                 .put("document_binary_url", document.links.binary.href)
                 .put("document_filename", document.originalDocumentName)
                 .build());
     }
+
+    @Test
+    void shouldToggleCtscCaseFlagOnWhenTheCtscFeatureIsEnabled() {
```

Comment:
I think the word toggle is misleasding here with the addition of launch darkly bits. Lets keep toggle to toggle service and then the ctsc case flag as yes no

Reply:
Changed to 'set'